src/Facade/ResourceBuilders/WorkstationResourceBuilder.cs

```diff
@@ -49,6 +52,11 @@ private IEnumerable<LinkResource> BuildLinks(Workstation model, IEnumerable<stri
             {
                 yield return new LinkResource { Rel = "self", Href = this.GetLocation(model) };
             }
+
+            if (this.authService.HasPermissionFor(AuthorisedActions.WorkstationAdmin, claims))
+            {
+                yield return new LinkResource { Rel = "workstation.admin", Href = "/stores2/work-stations/admin" };
```

Comment:
This link rel isn't quite RESTful enough... hypermedia links need to be meaningful, i.e. the rel should explain what the link lets you do - so on this case (assuming your two related actions are creating and updating workstations) there should be two link rels

rel: create
and 
rel: update

and the link should be an actual endpoint your api accommodates (/stores2/work-stations/admin isn't a real url where anything lives)

so here the hrefs should be the url that you make the request to create to (/stores2/workstations) or update to (/stores2/workstations/< id-of-thing-you-are-updating>) in each case

as an aside, is one workstation one word or two? if one i'd change the urls from /work-stations to /workstations